opg-wordpress-1604/tests/controls/default.rb

```diff
@@ -17,6 +17,14 @@
     its('stdout') { should match %r{64M} }
   end
 
+  impact 1
+  title 'Wordpress can be embedded'
+  desc 'Ensure Wordpress can be embedded via Frame Options'
+  describe command('grep -ir X-Frame-Options /etc/nginx/conf.d/app.conf') do
+    # X-Frame-Options should not be found.
```

Comment:
This should be in the description, but we can tidy it up later on.

Reply:
What should it say?